backends/fluidreview.py

```diff
@@ -0,0 +1,136 @@
+"""
+FluidReview API backend
+"""
+from urllib.parse import urljoin
+from datetime import timedelta
+from django.conf import settings
+from requests_oauthlib import OAuth2Session
+
+from backends.models import OAuthToken
+from backends.utils import utc_now
+
+API_NAME = 'FluidReview'
+API_BASE_URL = 'https://bootcampmit.fluidreview.com/api/v2/'
```

Comment:
Can you make this an environment variable and pass it into FluidReviewAPI?

Reply:
It seems these are unlikely to change (except rarely for the version `v2`), and will always be the same in CI, RC, production?

Reply:
The production instance is `mitbootcamps`, I'm not sure what `bootcampmit` is specifically. I think it's possible we could have two in the future, a testing instance and a production one. In any case I think this piece of the URL should be configurable so that we don't need a code change if we do end up needing to change it

Reply:
OK, makes sense